tests/manager/models/test_etcd.py

```diff
@@ -23,42 +22,20 @@
 
 
 @pytest.fixture(scope="module")
-def client() -> Client:
-    return Client(Schema(query=Queries, mutation=Mutations, auto_camelcase=False))
-
-
-@pytest.fixture(scope="module")
-def context(etcd_container) -> GraphQueryContext:  # noqa: F811
+def etcd_context(base_context, etcd_container) -> GraphQueryContext:  # noqa: F811
```

Comment:
There is no need to understand the **backend** or **detailed implementation**, whether it's `etcd` or something else. Therefore, I recommend keeping the name `context` as it is.
```suggestion
def context(base_context, etcd_container) -> GraphQueryContext:  # noqa: F811
```